docs/csharp/tutorials/working-with-linq.md

```diff
@@ -80,64 +77,97 @@ static IEnumerable<string> Ranks()
     yield return "ace";
 }
 ```
+Place these underneath the `Main` method in your `Program.cs` file. These two methods both utilize the `yield return` syntax to produce a sequence as they run. The compiler builds an object that implements `IEnumerable<T>` and generates the sequence of strings as they are requested.
```

Comment:
I like that you've added links to related docs (iterator, extension methods). You even can go further and add links to API docs as well (so called, xref-links). For example, replace `IEnumerable<T>` with `System.Collections.Generic.IEnumerable%601` (without backticks) to generate the link to [`IEnumerable<T>`](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerable-1?view=netframework-4.7.2)

Reply:
Sure, I can definitely make this happen